framework/wazuh/configuration.py

```diff
@@ -723,3 +704,62 @@ def upload_group_file(group_id, tmp_file, file_name='agent.conf'):
         return upload_group_configuration(group_id, file_data)
     else:
         raise WazuhException(1111)
+
+
+def get_active_configuration(agent_id, component, configuration):
+    """
+    Reads agent loaded configuration in memory
+    """
+    if not component or not configuration:
+        raise WazuhException(1307)
+
+    sockets_path = common.ossec_path + "/queue/ossec/"
```

Comment:
Better use os.path.join :)

Reply:
Fixed in https://github.com/wazuh/wazuh/pull/2993/commits/545fa149c9ff1c9dd31143dcaeca17d459d91861